policy/modules/contrib/rhcd.fc

```diff
@@ -0,0 +1,6 @@
+/usr/bin/rhc			--	gen_context(system_u:object_r:rhcd_exec_t,s0)
```

Comment:
Does the cli command also require private type?

Reply:
removed, thanks